### PR TITLE
Add project information to pyproject.toml + run tests with python 3.11

### PR DESCRIPTION
**Pull request by HelgeKrueger**
The additional information should improve what is shown on pypi, e.g. https://pypi.org/project/vodozemac/
https://pypi.org/project/vodozemac-python/

Please republish the project to it with this description

### CODE REVIEW

File: python/pyproject.toml [hunk 1]
```diff
@@ -5,3 +5,12 @@ build-backend = "maturin"
 [tool.maturin]
 name = "vodozemac"
 bindings = "pyo3"
+
+[project]
+name = "vodozemac"
+description = "Python bindings for vodozemax"
```
Comment (poljar) [inline python/pyproject.toml]:
```suggestion
description = "Python bindings for vodozemac"
```

File: python/pyproject.toml [hunk 2]
```diff
@@ -5,3 +5,12 @@ build-backend = "maturin"
...
+readme = "../README.md"
+license = {file = "../LICENSE"}
+
+[project.urls]
+repository = "https://github.com/matrix-org/vodozemac-bindings"
```
Comment (poljar) [inline python/pyproject.toml]: There's a missing newline.

**Review by poljar — CHANGES_REQUESTED** [review]
Left a couple of nits, looks good otherwise.

Please note that I mostly published the bindings to reserve the name, they are still undocumented and some API breakage is planned.

**Comment by HelgeKrueger** [comment]
I force pushed the two changes.

**Comment by poljar** [comment]
This has been included in the new repo: https://github.com/matrix-nio/vodozemac-python/pull/5

**Pull request closed without merging**